### DISCUSSION TITLE
Max Cores and RAM

### DISCUSSION THREAD

**Discussion by dennyxx82** [OpenRadioss/OpenRadioss#1577] [Users]
how many cores and max ram can openradioss handle

**Comment by MarianBulla** [comment]
Hello dennyxx82,
first, thank you for your interest on OpenRadioss.
Regarding your question, in general, there is no limitation on the number of cores. 
The scalability in OpenRadioss is extremly good, but it depends on the size of your model (it does not make sense to use 1000 cores on an 100 elements model). We usually count few thousands elepents per core as a rough rule.
For ram, it also depends on the size of your model, but is much less compared to implicite solvers. There is also no upper limit to it.
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by dennyxx82** [reply]
thanks it is very clear. I asked why so many companies that sell commercial software insist that openradioss has many limitations, it can't use more than 2 cores and it's very slow.

**Reply by MarianBulla** [reply]
This is not true! Do the proof :-)
And try it by yourself. 
OpenRadioss is very fast and highly scalable. You can use as many cores as you want.
You just need IntelMPI or OpenMPI, the you can run it on very large clusters with thousands of CPUs.
Hope it helps :-)
Thanks and best regards,
Marian

**Reply by dennyxx82** [reply]
thank's.
